tce/src/base/bem/BEMTester.cc

```diff
@@ -73,38 +73,37 @@ BEMTester::canAddComponentEncoding(
 
     int socketEncodings = field.socketEncodingCount();
     for (int i = 0; i < socketEncodings; i++) {
-	SocketEncoding& existingEnc = field.socketEncoding(i);
+        SocketEncoding& existingEnc = field.socketEncoding(i);
         int alignment = calculateAlignment(
             encodingWidth, existingEnc.socketIDWidth(), field);
-        int commonBits =
-	    commonBitCount(
-		encoding, extraBits, existingEnc.encoding(),
-		existingEnc.extraBits(), alignment);
-	if (commonBits == static_cast<int>(MathTools::bitLength(encoding))
-            + static_cast<int>(extraBits) || 
-            commonBits == existingEnc.socketIDWidth()) {
-	    return false;
-	}
+        int commonBits = commonBitCount(
+            encoding, extraBits, existingEnc.encoding(),
+            existingEnc.extraBits(), alignment);
+        if (commonBits == static_cast<int>(MathTools::bitLength(encoding))
+                + static_cast<int>(extraBits) || 
+                commonBits == existingEnc.socketIDWidth()) {
```

Comment:
`commonBits == existingEnc.socketIDWidth()` should be on same indentation level as the first `commonBits == static...`.